ros/wavemap_ros/app/rosbag_processor.cc

```diff
@@ -20,41 +21,46 @@ int main(int argc, char** argv) {
   // Setup the wavemap server node
   ros::NodeHandle nh;
   ros::NodeHandle nh_private("~");
-  wavemap::WavemapServer wavemap_server(nh, nh_private);
+  WavemapServer wavemap_server(nh, nh_private);
 
   // Read the required ROS params
   std::string rosbag_paths_str;
   nh_private.param("rosbag_path", rosbag_paths_str, rosbag_paths_str);
   std::string input_pointcloud_republishing_topic;
 
   // Create the rosbag processor and load the rosbags
-  wavemap::RosbagProcessor rosbag_processor;
+  RosbagProcessor rosbag_processor;
   std::istringstream rosbag_paths_ss(rosbag_paths_str);
   if (!rosbag_processor.addRosbags(rosbag_paths_ss)) {
     return -1;
```

Comment:
Maybe give the user some kind of error? Or does addRosbag already output something?

Reply:
addRosbags will already throw an error if it cannot open one of the bags